---
template: blog_post.html
description: 'Zurich Seminars in Bioinformatics - Lennart Opitz'
authors:
  - '@mbaudis'
date: 2020-07-09
---


### HiDRA-seq: High-Throughput SARS-CoV-2 Detection by RNA Barcoding and Amplicon Sequencing

* 12:00 in Y-55L-06, UZH Irchel Campus and 12:15 om  ZOOM Call

<!--more-->

__Abstract__ The recent outbreak of a new coronavirus that causes a Severe Acute Respiratory Syndrome in humans (SARS-CoV-2) has developed into a global pandemic with over 11 million reported cases and more than 530,000 deaths worldwide. Many countries have faced a shortage of diagnostic kits as well as a lack of infrastructure to perform necessary testing. Due to these limiting factors, only patients showing symptoms indicating infection were subjected to testing, whilst asymptomatic individuals, who are widely believed to be responsible for the fast dispersion of the virus, were largely omitted from the testing. The inability to implement population diagnostic and contact tracing strategies has forced many countries to institute lockdowns with severe economic and social consequences. The World Health Organization (WHO) encouraged affected countries to increase testing capabilities to identify new cases, allow for a well-controlled lifting of lockdown measures, and be ready for future outbreaks. Here, we propose HiDRA-seq, a rapidly implementable, high throughput, and scalable solution that uses NGS lab infrastructure and reagents for population SARS-CoV-2 testing. This method is based on the use of indexed oligo-dT primers to generate barcoded cDNA that can be multiplexed from a large number of patient samples. The protocol targets a region of the putative ORF10, which is highly conserved in the different SARS-CoV-2 isolates sequenced to date. Since the targeted region is located near the 3â€™-end of the genome, HiDRA-seq can capture both the viral genomic RNA (gRNA) as well as all subgenomic RNA transcripts (sgRNA) generated in infected cells.

HiDRA-seq entails a viral enrichment, followed by a small amount of short read sequencing data and a basic bioinformatics pipeline for downstream mapping and diagnosis. The small amount of short read sequencing data required allows the multiplexing of hundreds to thousands of patients in one sequencing run and is an affordable reality. Furthermore, HiDRA-seq can be adapted by a wide variety of short read sequencers with diverse outputs across the clinics. The nature of the enrichment step and detection of both gRNA and sgRNA in this protocol offers incredible versatility, as it can be tailored to any other respiratory virus or organism of interest that produces poly-adenylated transcripts. The implementation of such a rapid and versatile approach such as HiDRA-seq would definitely enable a more efficient outbreak management. 
