---
template: post.html
title: Frequent co-regulation of splicing and polyadenylation by RNA-binding proteins inferred with MAPP
description: 'Zurich Seminars in Bioinformatics - Maciek Bak | Biozentrum, University of Basel | Swiss Institute of Bioinformatics'
date: 2022-03-24
authors:
  - '@mbaudis'
---

* 12:15 UZH Irchel Y55-l-06/08 and ZOOM Call

The processing of nascent pre-mRNA consists of many steps, where splicing and primary transcript polyadenylation play key roles in determining transcriptome and subsequently, proteome diversity. Several studies indicate that many RNA-binding proteins (RBPs) act both on splicing as well as 3’ end processing but the context of this multi-level regulation and the full spectrum of RBPs involved are yet to be discovered. To facilitate answering these questions we have developed a novel computational method to identify RBPs that could shape the pre-mRNA maturation.
<!--more-->
The method aims to explain both the inclusion of ‘cassette’ exons as well as the relative usage of distinct poly(A) sites, quantified from RNA-seq data, in terms of the activity of short sequence motifs in the proximity of splice sites and polyadenylation sites. By scanning the neighborhood of processing sites with a sliding window the method is able to uncover positional signals thus providing a fine-grained resolution into the action of potential regulators. The tool includes visualizations in terms of  ‘impact maps’ which additionally provide insight into whether the RBP acts in an activatory/inhibitory manner. Applying our approach to RBP knock-down experiments we were able to identify both novel and well-known regulators of splicing and/or polyadenylation. Analyzing data from The Cancer Genome Atlas we successfully recovered a previously reported regulator in glioblastoma, and we further uncovered sequence motifs that are associated with RNA processing in other conditions.