---
title: 'ZH Seminars Bioinformatics - Helena L. Crowell'
date: 2019-06-05
layout: default
author: 'Helena L. Crowell'
excerpt_separator: <!--more-->
www_link:
www_links_formatted:
pdf_file_name:
category:
  - events
tags:
  - seminars
  - BIO612
---

## {{ page.title }}
### On the discovery of population-specific state transitions<br>from single-cell RNA sequencing data

* Wednesday, 2019-04-03
* 12:15 in Y-55L-12, UZH Irchel campus

<!--more-->

*Abstract* Single-cell RNA-sequencing (scRNA-seq) has quickly become an empowering technology to profile the transcriptomes of individual cells on a large scale. Many early analyses of differential expression have aimed at identifying differences between cell types (or clusters), and thus are focused on finding markers for cell populations (experimental units are cells).  
Given the emergence of replicated multi-condition scRNA-seq datasets, a new area of focus has become making sample-level inferences, known as Differential State (DS) analysis, e.g., condition-specific responses of specific immune cell subsets. DS analysis: i) should be able to detect “diluted” changes that only affect a single cell type or a subset of cell types; and, ii) is intended to be orthogonal to clustering or cell type assignment.  
We surveyed the methods available to perform DS analyses, including cell-level mixed models and models on aggregated *pseudobulk* data, developed a simulation that mimics single and multi-sample scRNA-seq data and provide robust tools for multi-sample multi-condition analysis within the muscat R package. In this talk, I will introduce the concept of DS analysis, present our simulation framework along with method comparison results obtained from a comprehensive simulation study, and demonstrate an application of our scRNA-seq analysis workflow to an exemplary biological dataset. 